pkg/iosrc/uri_windows_test.go

```diff
@@ -0,0 +1,16 @@
+package iosrc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestURIWin(t *testing.T) {
+	expected := `c:\test\folder`
+	uri, err := ParseURI(expected)
+	require.NoError(t, err)
+	assert.Equal(t, "file:///c:/test/folder", uri.String())
+	assert.Equal(t, expected, uri.Filepath())
+}
```

Comment:
Can you add a test that parses `a\b\..\c` and make sure it gives the expected uri?

Reply:
Good idea.